securedrop/tests/test_source.py

```diff
@@ -28,6 +28,11 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_page_not_found(self):
+        response = self.client.get('/UNKNOWN')
+        self.assertEqual(response.status_code, 404)
+        self.assertIn("locate what you requested", response.data)
```

Comment:
I would prefer this include the full custom error message ("Sorry, we couldn't locate what you requested."), so it's clearer exactly what this test is trying to achieve: that is, that a 404 error results in a response that includes the content of our custom 404 error template (`notfound.html`). It also might be worth adding a comment or docstring to help clarify the intent of this test.